<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ARM_OD_view</class>
 <widget class="QMainWindow" name="ARM_OD_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="mainTabWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="actionOpen_Atlas"/>
    <addaction name="actionOpen_from_file"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionKoordinatometriyaParameters"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMap"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen_Atlas">
   <property name="text">
    <string>Open Atlas</string>
   </property>
  </action>
  <action name="actionOpen_from_file">
   <property name="text">
    <string>Open from file</string>
   </property>
  </action>
  <action name="actionKoordinatometriyaParameters">
   <property name="text">
    <string>Koordinatometriya parameters</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
